cypress/support/fragments/topMenu.js

```diff
@@ -24,5 +24,6 @@ export default {
   checkInPath: '/checkin',
   checkOutPath: '/checkout',
   receivingPath: '/receiving',
-  bulkEditPath: '/bulk-edit'
+  bulkEditPath: '/bulk-edit',
```

Comment:
so editing this file is obsolete in this pull request

Reply:
Done